lib/download.test.ts

```diff
@@ -64,7 +66,7 @@ describe('sane downloads', () => {
 	});
 });
 
-describe('fetchTargetInferredVersion', () => {
+describe.skip('fetchTargetInferredVersion', () => {
```

Comment:
Local testing leaked ? should be removed.

Reply:
Thanks!